conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/RemoteExceptionMapper.java

```diff
@@ -54,64 +54,35 @@
 
     @Override
     public Response toResponseInner(RemoteException exception) {
-        ErrorType errorType = mapErrorType(exception);
-        Response.ResponseBuilder builder = Response.status(errorType.httpErrorCode());
-
-        try {
-            SerializableError error = SerializableError.builder()
-                    .errorName(errorType.name())
-                    .errorCode(errorType.code().toString())
-                    .errorInstanceId(exception.getError().errorInstanceId())
-                    .build();
-            builder.type(MediaType.APPLICATION_JSON);
-            builder.entity(error);
-        } catch (RuntimeException e) {
-            log.warn(
-                    "Unable to translate exception to json",
-                    SafeArg.of("errorInstanceId", exception.getError().errorInstanceId()),
-                    SafeArg.of("errorName", exception.getError().errorName()),
-                    e);
-            // simply write out the exception message
-            builder.type(MediaType.TEXT_PLAIN);
-            builder.entity("Unable to translate exception to json. Refer to the server logs with this errorInstanceId: "
-                    + exception.getError().errorInstanceId());
-        }
-        return builder.build();
-    }
-
-    private ErrorType mapErrorType(RemoteException exception) {
-        Status status = Status.fromStatusCode(exception.getStatus());
-
-        if (status.getStatusCode() == 401) {
-            log.info(
-                    "Encountered a remote unauthorized exception."
```

Comment:
I'd prefer to keep around the more specific log lines based on status code. Makes it easier to sift through the data, and decouples their log levels.

Reply:
We already log the status as an arg here, shouldn't that be sufficient to filter logs?

Reply:
It should be, although filtering on args only seems to work intermittently for me. Numeric value handling is wonky due to the floating point coercion. Not a strong opinion on this, just what I would have done.